src/features/theme/themes/fuwari/styles/index.css

```
@@ -0,0 +1,258 @@
+/* Theme: fuwari - Fuwari-style variables and utilities */
+@import "tailwindcss";
+@import "../../../../../styles/shared.css";
+@import "./markdown.css";
+@plugin "@tailwindcss/typography";
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)